snuba/clickhouse/formatter/expression.py

```diff
@@ -212,16 +217,7 @@ def _format_string_literal(self, exp: Literal) -> str:
         return "$S"
 
     def _format_number_literal(self, exp: Literal) -> str:
-        return "$N"
-
-    def _format_boolean_literal(self, exp: Literal) -> str:
-        return "$B"
-
-    def _format_datetime_literal(self, exp: Literal) -> str:
-        return "$DT"
-
-    def _format_date_literal(self, exp: Literal) -> str:
-        return "$D"
```

Comment:
this diff is misleading, it looks like ` return str(_gen_random_number())` is the implementation for `format_date_literal` but I just removed `format_date_literal` and made it inherit form the non-anonymized clickhouse expression formatter.

Reply:
I may have missed something but do we want anonymized queries to be runnable? 

Because if we don't care about that it seems to me like it may be more clear to see a $N rather than an arbitrary number (like 4) that you're looking at an anonymized value.

Reply:
Short answer is: yes we do want to be able to run anonymized queries. 

The reason is that a lot of times the reason you're looking at a query is:

* Clickhouse said it was invalid
* It timed out for some reason

When that happens, I want to either:

* Copy and paste it into my local clickhouse and see if the error is reproducible
* Run an EXPLAIN on it and see what it gives me

Either way I want to be able to run it